<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="plotter" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>139</y>
      <width>661</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGet">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>356</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>getData</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>291</width>
      <height>141</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>192.168.0.18</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>311</width>
      <height>85</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Ip do Servidor</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lineIP">
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="connectButton">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="disconnectButoon">
           <property name="text">
            <string>Disconnect</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>timing</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QSlider" name="sliderTiming">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="startButton">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="stopButton">
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdTiming">
       <property name="intValue" stdset="0">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>675</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>plotter</class>
   <extends>QWidget</extends>
   <header>plotter.h</header>
   <container>1</container>
   <slots>
    <slot>recebe(float)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>lineIP</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderTiming</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdTiming</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
